srcpkgs/dmenu/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'dmenu'
 pkgname=dmenu
-version=4.8
+version=4.9
 revision=1
 makedepends="libXinerama-devel libXft-devel freetype-devel"
 short_desc="A generic menu for X"
```

Comment:
Please remove the article 'A'

Reply:
New patch had been pushed. I think it's better now.